code/dynamic_programming/src/matrix_chain_multiplication/matrix_chain_multiplication.cpp

```diff
@@ -40,15 +43,36 @@ int MatrixChainOrder(int p[], int n)
  
     return m[1][n-1];
 }
- 
+/*Recursive Approach Solution
+ *Time Complexity > O(2^n)
+ */
+int recursiveMatrixChainOrder(const int p[], int i, int j)
+{
+    if (i==j)
+        return 0;
+    int q=INT_MAX;
+    int temp;
+    /* q = Minimum number of scalar multiplications needed
+       to compute the matrix A[i]A[i+1]...A[j] = A[i..j] where
+       dimension of A[i] is p[i-1] x p[i] */
+
+    for(int k=i;k<j;k++)
+    {   
+        temp=recursiveMatrixChainOrder(p,i,k)+recursiveMatrixChainOrder(p,k+1,j)+p[i-1]*p[j]*p[k];
```

Comment:
And other lines, please check the spaces and width limitation of the coding style guide.